### DISCUSSION TITLE
Using has- in combination with &> child selector

### DISCUSSION THREAD

**Discussion by etturD4** [tailwindlabs/tailwindcss#13004] [Help] [answered]
I have following HTML

```
    <div className='has-[.true]:[&>*]:hidden px-1 py-2 text-sm border-t relative md:border-r'>
          <MenuToggle />
          <div
            id='menuContainer'
            className='flex flex-row items-center w-full h-full md:flex-col'
          >
```

Inside MenuToggle there is logic to add .true class and that works with the parent's has- selector. For instance, this works 

`    has-[.true]:hidden`

this will hide the parent. However, I do not wish to hide parent, but another child of parent, the one with id=menuContainer. So I tried to make child selector work, via [&>*], but I'm missing something.

something like this is what I am trying to do, but failing

`    has-[.true]:[&>#menuContainer]:hidden`

if this is not possible, maybe I can restructure tags somewhat and make it work with peers

**Comment by etturD4** [comment]
I got this to work using peer, however, if it is possible to combine has with child selector, I would appreciate an example .

**Reply by wongjn** [reply, marked as answer]
https://play.tailwindcss.com/C4a70jbe28

**Reply by etturD4** [reply]
Thanks, its good to know! But looking at the code, peer solution is much simpler and cleaner.
